Jenkinsfile_CNP

```diff
@@ -21,6 +21,19 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 }
 
 withPipeline("nodejs", product, component) {
+  afterCheckout {
+    onMaster {
+      try {
+        sh """
+          git fetch origin demo:demo
```

Comment:
Do we need to even fetch demo here?

Reply:
Yes, we need to. Jenkins wants to avoid unnecessary network traffic and fetches only the build branch and master, so it has no knowledge of demo until we fetch it explicitly.